Lib/test/test_curses.py

```diff
@@ -447,6 +446,22 @@ def test_update_lines_cols(self):
         # can be called.
         curses.update_lines_cols()
 
+    @requires_curses_func('ncurses_version')
+    def test_ncurses_version(self):
+        v = curses.ncurses_version
+        self.assertIsInstance(v[:], tuple)
+        self.assertEqual(len(v), 3)
+        self.assertIsInstance(v[0], int)
+        self.assertIsInstance(v[1], int)
+        self.assertIsInstance(v[2], int)
+        self.assertIsInstance(v.major, int)
+        self.assertIsInstance(v.minor, int)
+        self.assertIsInstance(v.patch, int)
+        self.assertEqual(v[0], v.major)
+        self.assertEqual(v[1], v.minor)
+        self.assertEqual(v[2], v.patch)
+        self.assertTrue(v > (0, 0, 0))
```

Comment:
I would prefer to check that each field is > 0.

```
self.assertGreater(v.major, 0)
self.assertGreater(v.minor, 0)
self.assertGreater(v.patch, 0)
```

``> (0, 0, 0)`` test is not enough:

```
>>> (1, -2, 3) > (0, 0, 0)
True
```

Reply:
v.minor is 0 in the actual version.